features/ProductNews.feature

```diff
@@ -0,0 +1,46 @@
+@owner:derek
+@author:lilla
+
+Feature: Docs Product News Page
+
+    Display the Product News Page with cards to navigate to news.
+
+    @motivating
+    Scenario: There are two filter options at the top of the news column to filter based on tags and dates
+        Given a user wants to filter the news feed
+        Then they can select from the following dropdown filter items for type
+
+            | postType         |
+            | All Posts        |
+            | Feature Releases |
+            | Monthly Updates  |
+            | Product Updates  |
+
+        And they can select from the following dropdown items for dates
+
+            | dates         |
+            | From All-time |
+            | Last 30 days  |
+            | Past Year     |
+
+    @motivating
+    Scenario: News cards are showing on the page in one column regardless of the screen size
+        Given a user looks at the card
+        Then they see the following items
+
+            | cardParts |
+            | Title     |
+            | Date      |
+            | Tags      |
+            | Content   |
+
+        And they see a 'CTA button' if the following settings are included in the contentful editor
```

Comment:
This is good detail around the CTA button but if the spec is about the news cards showing in one column, this information is better placed in your next spec